modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for some connections within {product-title-short} and between {product-title-short} and external components are shown in the following table. You might need this information for configuring the firewalls in your system to allow traffic on certain ports.
+
+.{product-title-short} connections between components
+
+[cols="4",options="header"]
+|===
+| Component or external entity | Connection type | Port | Additional information
+
+| Central and Sensor on secured cluster
+a| * TCP/HTTPS
+* gRPC
+| 443
+| Sensor and Central primarily communicate over a bidirectional gRPC stream, initiated by Sensor to Central's port 443.
+
+
+| Central and user ({product-title-short} web portal or REST API)
+| HTTPS or gRPC
+| 443
+| The user reaches out to Central on port 443 by using the web portal or the REST API.
+
+| Central and user (CLI)
+| gRPC
+| 443
+|
+
+| Central and vulnerability feeds
+| HTTPS
+| 443
+|
+
+| Scanner to Scanner-DB
+| TCP
+| 5432
+| Scanner reaches out to Scanner DB at port 5432.
+
+| Scanner V4 Indexer and Central
+|
+|
+| Protocol and port used depends on the context.
+
+Scanner V4 Indexer to Scanner V4 DB
```

Comment:
I have noticed on the rendered page broken formatting. Maybe this line causes problems.
```suggestion
| Scanner V4 Indexer to Scanner V4 DB
```

Besides that. 
- Since we are already listing `Scanner -> Scanner DB` connection, we should also add `Central -> Central DB`.
- Also `Sensor -> Scanner` connection is not mentioned here.

Reply:
@kcarmichael08 - it should be like this:
- `Central -> Central DB` - should be the same as `Scanner -> Scanner DB` (TCP, 5432). btw. This can be different if a customer uses "bring your own DB". https://docs.openshift.com/acs/4.5/installing/installing_ocp/install-central-ocp.html#install-central-operator-external-db_install-central-ocp

- `Sensor -> Scanner` - should be the same as `Central -> Scanner` (maybe this can be grouped together). Probably @RTann knows better how `Sensor` "talks" with `Scanner`.